Flepper.QueryBuilder/Operators/Comparison/ComparisonOperators.cs

```diff
@@ -12,6 +12,11 @@ public ComparisonOperators(StringBuilder command) : base(command)
 
         public IComparisonOperators EqualTo(object value)
         {
+            if(value == null)
+            {
+                Command.AppendFormat("IS NULL ");
+                return this;
```

Comment:
Instead of returning this, do an if - else

```csharp
if (value is null) 
    Command.AppendFormat("IS NULL "); 
else
    Command.AppendFormat("= {0} ", value.InsertQuotationMarksIfDateOrString()); 
```